src/CallHandler.tsx

```diff
@@ -356,6 +356,7 @@ export default class CallHandler {
                     this.play(AudioID.Ringback);
                     break;
                 case CallState.Ended:
+                {
```

Comment:
any reason these were added?

Reply:
I added a variable declaration which can't be done in a switch case statement without its own block